.gitignore

```diff
@@ -1,3 +1,4 @@
+Gemfile.lock
```

Comment:
@bufferoverflow can you provide references that mention that committing the `.lock` file is bad practice?